src/app/components/crypto-assets/bitcoin/runes-asset-list/runes-asset-item.layout.tsx

```diff
@@ -27,7 +27,7 @@ export function RunesAssetItemLayout({ rune }: RunesAssetItemLayoutProps) {
             side="left"
           >
             <styled.span data-testid={rune.rune_name} fontWeight={500} textStyle="label.02">
-              {formattedBalance.value} {rune.symbol}
+              {formattedBalance.value} {rune.symbol ?? '¤'}
```

Comment:
Would extract as a const somewhere, as we'll likely use again

Reply:
Moved it to when we make the type after the query.